src/pages/List/ListContents.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import ListCard from './ListCard';
+import ListOption from './ListOption';
+
+function ListContents(props) {
```

Comment:
- props를 인자로 받는 부분에서 바로 구조분해 할당해서 사용해 주세요!
```suggestion
function ListContents({ inputQuery, data }) {
```